Dockerfile

```diff
@@ -22,6 +22,9 @@ RUN /bin/bash -e /security_updates.sh && \
     apt-get install -yqq --no-install-recommends \
         nginx-light \
     && \
+    apt-get install -yqq --no-install-recommends \
```

Comment:
this installation can be combined right below `nginx-light`, no need for an additional install line